html-css/Task3/index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>Sitename - Description</title>
+        <meta name="description" content="description">
+        <meta name="keywords" content="site, web, sitename">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="css/style.css" rel="stylesheet">
+        <link href="css/page.css" rel="stylesheet">
+        <link href="css/menu.css" rel="stylesheet">
+        <link href="css/banner.css" rel="stylesheet">
+        <!--<link href="css/article.css" rel="stylesheet">-->
+        <link href="css/info.css" rel="stylesheet">
+        <link href="css/footer.css" rel="stylesheet">
+        <link href="css/subblock.css" rel="stylesheet">
+        <link href="css/header.css" rel="stylesheet">
+        <link href="css/contact.css" rel="stylesheet">
+        <link href="css/content.css" rel="stylesheet">
+        <link href="css/main.css" rel="stylesheet">
+        <link href="css/fonts.css" rel="stylesheet">
+        <link href="css/font-awesome.min.css" rel="stylesheet">
+        <link href="css/article.less" rel="stylesheet/less" type="text/css" />
+        <script src="js/less.min.js" ></script>
```

Comment:
Typically, developers perform compilation less -> css on server side (using grunt, gulp, bundles, etc) and include compiled and minified css files to the page.